### ISSUE TITLE
3.7 advanced_install guide should include a section on enabling Gluster Block storage

### ISSUE THREAD

**Issue by etsauer** [openshift/openshift-docs#8534]
### Which section(s) is the issue in?

The [Configuring Ansible](https://docs.openshift.com/container-platform/3.7/install_config/install/advanced_install.html#configuring-ansible) section of the 3.7 and later install guides.

### What needs fixing?

We need to add a section instructing on the options for configuring Gluster Block storage. Probably a section below [Configuring GlusterFS Persistent Storage](https://docs.openshift.com/container-platform/3.7/install_config/install/advanced_install.html#advanced-install-glusterfs-persistent-storage) called "Configuring GlusterBlock Persistent Storage" that highlights the additional ansible variables and verification steps needed to enable block storage in addition to GlusterFS. An Example variable set would be:

```
openshift_storage_glusterfs_block_deploy: true
openshift_storage_glusterfs_block_host_vol_create: true
openshift_storage_glusterfs_block_host_vol_size: 50
openshift_storage_glusterfs_block_host_vol_max: 10
openshift_storage_glusterfs_block_storageclass: true
openshift_storage_glusterfs_block_storageclass_default: false
```

**Comment by vikram-redhat** [comment]
@jarrpa do your recent changes solve this?

**Comment by jarrpa** [comment]
The changes made to the enterprise-3.9 branch go into some detail about the glusterblock related variables, and provide a pointer to the full list of variables for controlling GlusterFS behavior. If this issue is specific for 3.7, then this does not solve the problem. That problem will be solved once the OCP 3.7 release of openshift-ansible has all recent changes backported, at which time the openshift-docs enterprise-3.7 branch will also be updated.

**Comment by vikram-redhat** [comment]
Thanks @jarrpa . I will then close this issue and if the problem persists when the backport is finished we can revisit.
